bloom/commands/git/import_upstream.py

```diff
@@ -38,6 +38,7 @@
 import tarfile
 import tempfile
 
+from io import open
```

Comment:
Sorry I commented before seeing [this comment ](https://github.com/ros-infrastructure/bloom/pull/432#discussion_r134864835), I guess in that case we can do the same here so that "open" always refers to the builtin `open` and `io.open` to the fancier version.